lib/util.js

```diff
@@ -504,49 +505,60 @@ const isArray = exports.isArray = Array.isArray;
 function isBoolean(arg) {
   return typeof arg === 'boolean';
 }
-exports.isBoolean = isBoolean;
+exports.isBoolean = deprecate(isBoolean
+  , 'util.is* functions are deprecated, please use a user-land alternative.');
```

Comment:
Maybe, I just copy-pasta'd these.

Also though, it may be worth messaging that all of these have been deprecated? I'm not so sure.